templates/lune/partials/components/chapter-name.snip.html

```diff
@@ -0,0 +1,3 @@
+{{#chapter}}
```

Comment:
Yeah so we have the chapter name, which sits on the left hand side, and the chapter numbers that sit on the right. We've split them up, as the chapter name is re-used on the store page, but the numbers are not. 

These are also included multiple times on the product pages, so it made sense to avoid duplication and create partials for them